HTTP_Engine/Compute/GetRequestAsync.cs

```diff
@@ -57,15 +57,15 @@ public static async Task<string> GetRequestAsync(string uri, HttpClient client =
 
         /***************************************************/
 
-        public static async Task<string> GetRequestAsync(string url, Dictionary<string, object> headers = null, Dictionary<string, object> parameters = null, HttpClient client = null)
+        private static async Task<string> GetRequestAsync(string url, Dictionary<string, object> headers = null, Dictionary<string, object> parameters = null, HttpClient client = null)
```

Comment:
Maybe I agree with these ones to be private for now. However, I would like to raise a discussion about the support of `async` components in the UI instead.